### PR TITLE
Technology

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
##Summary
add technology

##Screenshot
captura de pantalla 2018-09-10 a la s 10 52 48

##Trello Card
https://trello.com/c/WVijNyRG/22-tecnologia

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,4 +10,9 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1>  
+  <div class="technoContent">
```
Comment (mmamaniwolox) [inline index.html]: All the elements inside this div miss a tab

File: index.html [hunk 2]
```diff
@@ -10,4 +10,9 @@
...
+  <img class="imgTechno" src="images/react_icon.svg">
```
Comment (mmamaniwolox) [inline index.html]: Add an alt to this image

File: index.html [hunk 3]
```diff
@@ -10,4 +10,9 @@
...
+  <p class="text">ReactJS<br><br> <strong>Release:2013</strong><br><br> React is a declarative,<br>component-based <br>Javascript library for<br> building user interfaces.</p>
```
Comment (mmamaniwolox) [inline index.html]: What is this line?! For bold use css `font-weight: bold`. For line break set a width to container.

File: scss/application.scss [hunk 4]
```diff
@@ -1,2 +1,3 @@
 @import 'equalizer';
 @import 'titles';
+@import 'tech';
```
Comment (mmamaniwolox) [inline scss/application.scss]: Add a tailing line

File: scss/tech.scss [hunk 5]
```diff
@@ -0,0 +1,40 @@
+ @import 'color';
+
+.technoContent {
+  margin: 12px;
+  background: $color-green;
+  height: 300px;
+  width: 175px;
+  padding: 8.5px;
```
Comment (mmamaniwolox) [inline scss/tech.scss]: Is this to center the content? Use flex `justify-content: center` to align horizontally and `align-items: center` to center vertically if thats what you want

File: scss/tech.scss [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  position: relative;
+  border-radius: 5px;
+  display: flex;
+  
+  &::before{
```
Comment (mmamaniwolox) [inline scss/tech.scss]: Space before curly braces. All its content miss a tab

File: scss/tech.scss [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  bottom: 5px;
+  right: 5px;
+  width: 175px;
+  border: 1px solid $color-green;
+  border-radius: 5px;
+  position: absolute; 
+  }
+}
+
+.text{
```
Comment (mmamaniwolox) [inline scss/tech.scss]: Space before curly braces

File: scss/tech.scss [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+  text-align: center;
+  flex-direction: row;
+  padding: 5px;
+  margin: 110px 0 0 0;
```
Comment (mmamaniwolox) [inline scss/tech.scss]: Margin-top ?

File: scss/tech.scss [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+  right: 38px;
+}
+
+.imgTechno{
```
Comment (mmamaniwolox) [inline scss/tech.scss]: Space before curly braces

File: scss/tech.scss [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+  text-align: center;
+  position: absolute;
+  right: 38px;
+  bottom: 187px;
```
Comment (mmamaniwolox) [inline scss/tech.scss]: This shouldn't be set as absolute position, you can just center it horizontally and space the content vertically

File: scss/titles.scss [hunk 11]
```diff
@@ -6,7 +6,7 @@
   position: relative;
   margin: 80px;
 
-  &:after {  
+  &:before {
```
Comment (mmamaniwolox) [inline scss/titles.scss]: &::before

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]
For all styles dont use camel case, use `techno-content` for example

File: index.html [hunk 12]
```diff
@@ -10,4 +10,11 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1>  
+    <div class="technoContent">
```
Comment (mmamaniwolox) [inline index.html]: Div childs miss tabbing

File: scss/tech.scss [hunk 13]
```diff
@@ -0,0 +1,45 @@
+ @import 'color';
+
+.technoContent {
```
Comment (mmamaniwolox) [inline scss/tech.scss]: Dont use cammel case!

File: scss/tech.scss [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+  height: 300px;
+  width: 175px;
+  position: relative;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+ 
+  &::before {
```
Comment (mmamaniwolox) [inline scss/tech.scss:15]: Add a tab to its childs

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]
Change cammel case style classes

**Review by mmamaniwolox — APPROVED** [review]

File: scss/tech.scss [hunk 15]
```diff
@@ -0,0 +1,45 @@
...
+    right: 5px;
+    width: 175px;
+    border: 1px solid $color-green;
+    border-radius: 5px;
+    position: absolute; 
+    z-index: -1;
+  }
+}
+
+.text-react {
```
Comment (mmamaniwolox) [inline scss/tech.scss]: You will have more technologies, and you will share the css classes. Try to use a more generic name

File: scss/tech.scss [hunk 16]
```diff
@@ -0,0 +1,45 @@
...
+  font-weight: bold; 
+}
+
+.text-complete {
+  width: 160px;
+}
+
+.img-techno {
+  width: 100px;
+  height: 100px;
```
Comment (mmamaniwolox) [inline scss/tech.scss]: Use max-height instead, and you dont need width, with height its enought

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by mmamaniwolox — APPROVED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**